repository-data/application/src/main/resources/hcm-config/namespaces/website/person.yaml

```diff
@@ -60,29 +55,34 @@ definitions:
             plugin.class: org.onehippo.forge.contentblocks.ContentBlocksFieldPlugin
             wicket.id: ${cluster.id}.field
             wicket.skin: skin/content-blocks.css
-          /clinician:
+          /SocialMediaBar:
```

Comment:
Is it meant to have a link to the social media bar?

Reply:
Removed SocialMediaBar